addOns/oast/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 - Interactsh: server URL change in Options deregisters old server URL and registers new server URL
 - OAST Interactsh Options Dialog: If host or token config changed the 'New Payload' Button generates the Payload still with the old config. 
 Button is disabled in that case.
+- Set HttpSender's Initiator to 'OAST_INITIATOR'
```

Comment:
- Better under Changed, not a fix.
 - Consider adding the value so the user does not have to check the unreleased code.